indico/modules/events/contributions/operations.py

```diff
@@ -73,8 +73,9 @@ def create_contribution(event, contrib_data, custom_fields_data=None, session_bl
     return contrib
 
 
+@make_interceptable
 @no_autoflush
-def update_contribution(contrib: Contribution, contrib_data: dict, custom_fields_data=None):
+def update_contribution(contrib: Contribution, contrib_data: dict, custom_fields_data=None, _extra_log_fields=None):
```

Comment:
```suggestion
def update_contribution(contrib: Contribution, contrib_data: dict, custom_fields_data=None, *, _extra_log_fields=None):
```

Let's make this keyword-only. It's already the case in `update_event` etc.